### PR TITLE
[feature] Add input grype-version

### PR DESCRIPTION
**Pull request by ViacheslavKudinov**
- Added `grype-version` input to have ability to set specific version of Grype. 
- Removed `debug` input related things (input doesn't exist on the current version of JS code but adds extra annotation on Action tab:

> "Unexpected input(s) 'debug', valid inputs are ['image', 'path', 'sbom', 'fail-build', 'output-format', 'severity-cutoff', 'only-fixed', 'add-cpes-if-none', 'by-cve']")

- Bump @babel/traverse from 7.22.8 to 7.23.4 (due to https://github.com/advisories/GHSA-67hx-6x53-jw92)

### CODE REVIEW

**Comment by ViacheslavKudinov** [comment]
Could be test of input has to be moved to some "demo" workflow, not sure about this moment.
Let me know if i have to move this part.

**Comment by ViacheslavKudinov** [comment]
Vi @kzantow 
Could you, please, take a look?

**Comment by ViacheslavKudinov** [comment]
Hello,
any news on approving/rejecting/etc this one?

**Comment by robfrank** [comment]
You can provide the version, it's just not documented (and you will get a warning):
```
- name: Anchore Scan API Image
      uses: anchore/scan-action@24fd7c9060f3c96848dd1929fac8d796fb5ae4b4 # v3.3.6
      id: scan
      with:
        fail-build: false
        output-format: sarif
        severity-cutoff: critical
        image: <YOUR IMAGE NAME>
        grype-version: v0.72.0
```

from here: https://github.com/anchore/scan-action/blob/6decf31b6b1e3287401988bb2cec00f2fa375127/dist/index.js#L23C1-L23C1

**Comment by ViacheslavKudinov** [comment]
Hi @willmurphyscode
is there any reaction from the "main" contributors?

**Comment by ViacheslavKudinov** [comment]
or @wagoodman ?

**Comment by ViacheslavKudinov** [comment]
Hi @willmurphyscode 
thank you for the answer.
When new release done, i can fix conflicts to get it resolved before review.

**Comment by willmurphyscode** [comment]
@ViacheslavKudinov good question! We removed snapshot testing for a few reasons:

1. This repo's job is to invoke grype, not test the output of grype. Snapshot tests on the output of grype's output belong in the grype repo.
2. Most changes to this repo are just bumping grype, which usually changes the output at least a bit, so we have to update snapshots on almost every change; having snapshot tests that need to change every time we bump grype just adds friction to keeping the repo up to date.

File: tests/sarif_output.test.js [hunk 1]
```diff
@@ -72,7 +72,11 @@ describe("SARIF", () => {
   it("debian", async () => {
     const sarif = await testSource(
       "localhost:5000/match-coverage/debian:latest",
+<<<<<<< HEAD
+      ["GHSA-9w8r-397f-prfh-Pygments", "GHSA-fp4w-jxhp-m23p-bundler"]
```
Comment (willmurphyscode) [inline tests/sarif_output.test.js:76]: Looks like this has a conflict

**Comment by ViacheslavKudinov** [comment]
I will merge GH Actions changes into my branch as well.

File: tests/sarif_output.test.js (hunk 1 above)
Comment (ViacheslavKudinov) [reply tests/sarif_output.test.js:76]: yep, not resolved all things.
Give me some time.
I will convert into Draft and when all is merged reopen Pr.

**Review by willmurphyscode — APPROVED** [review]

**Comment by willmurphyscode** [comment]
Thanks @ViacheslavKudinov! We'll get this out in the next release.

**Comment by ViacheslavKudinov** [comment]
Thank you @willmurphyscode

**Pull request merged**